javascript/ql/src/semmle/javascript/dataflow/TaintTracking.qll

```diff
@@ -643,6 +643,50 @@ module TaintTracking {
 
   }
 
+  /** 
+   * A check of the form `if(whitelist.indexOf(x) >= 0)`, which sanitizes `x` in its "then" branch. 
+   *
+   * Similar relational checks are also supported.
+   */
+  class RelationalIndexOfSanitizer extends AdditionalSanitizerGuardNode, DataFlow::ValueNode {
```

Comment:
I assume we can make this `private`?

Reply:
Yep.
I have also privatized the other new `indexOf` sanitizer.